src/lib/kdump/clients/finish.rb

```diff
@@ -0,0 +1,56 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+
+require "installation/finish_client"
+
+module Y2Kdump
+  module Clients
+    class Finish < ::Installation::FinishClient
+      def initialize
+        textdomain "kdump"
+
+        Yast.import "Kdump"
+        Yast.import "Mode"
+        Yast.import "Progress"
+      end
+
+      def title
+        # progress step title
+        _("Saving kdump configuration...")
+      end
+
+      def modes
+        [:installation, :update, :autoinst]
+      end
+
+      def steps
+        3
+      end
+
+      def write
+        progress_orig = Progress.set(false)
+        # propose settings for kdump
+        # if autoyast doesn't include settings for yast2-kdump
+        Kdump.Propose if !Kdump.import_called && (Mode.autoinst || Mode.autoupgrade)
```

Comment:
Ah, I was wrong, it is in SP2: https://github.com/yast/yast-yast2/commit/3a6987c12efb12fe6a8e5605b897f1d9b04ef0d8